src/src/AdminBundle/Command/AddMockedDataCommand.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace AdminBundle\Command;
+
+use AdminBundle\Entity\Admin;
+use AdminBundle\Entity\Attendee;
+use AdminBundle\Entity\Event;
+use AdminBundle\Entity\EventType;
+use AdminBundle\Entity\Language;
+use AdminBundle\Entity\Organizer;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class AddMockedDataCommand extends ContainerAwareCommand
+{
+    private function randomFirstName()
+    {
+        $names = array(
+            'Abdullah', 'Itzel', 'Franklin', 'Maci', 'Sierra', 'Nasir', 'Vivian', 'Jaxson',
+            'Reece', 'Sofia', 'Lawson', 'Jax', 'Gaige', 'Alec', 'Nyasia', 'Yesenia', 'Branden',
+            'Damon', 'Leia', 'Evelin', 'Virginia', 'Marisol', 'Charlee', 'Phoenix', 'Natasha',
+            'Valentino', 'Bryanna', 'Jazmin', 'Craig', 'Jermaine', 'Jaslene', 'Raina', 'Evangeline',
+            'Adalyn', 'Avery', 'Elianna', 'August', 'Lisa', 'Arthur', 'Amirah', 'Kenneth', 'Kiana',
+            'Marlee', 'Messiah', 'Teagan', 'Ishaan', 'Adyson', 'Justin', 'Ross', 'Janessa',
+        );
+
+        return $names[mt_rand(0, sizeof($names) - 1)];
+    }
+
+    private function randomLastName()
+    {
+        $surnames = array(
+            'Franco', 'Jordan', 'Griffith', 'Davies', 'Barrera', 'Goodwin', 'Bates', 'Deleon',
+            'Key', 'Walter', 'Gilmore', 'Barrett', 'Roberts', 'Wells', 'Durham', 'Lane', 'Lutz',
+            'Mccullough', 'Cobb', 'Banks', 'Conner', 'Cowan', 'Cortez', 'Villegas', 'Cooper',
+            'Brown', 'Knight', 'Newton', 'Allison', 'Sullivan', 'Ibarra', 'Glass', 'Downs',
+            'Coleman', 'Wilkins', 'Webster', 'Dickerson', 'Rivas', 'Fitzpatrick', 'Michael',
+            'Mccarty', 'Sutton', 'Leon', 'Cisneros', 'Aguilar', 'Chandler', 'Jennings', 'Sparks',
+            'Frye', 'Hooper',
+        );
+
+        return $surnames[mt_rand(0, sizeof($surnames) - 1)];
+    }
+
+    private function randomDate()
+    {
+        $random = new \DateTime();
+        $random->setTimestamp(mt_rand(1202055681, 1282055681));
+
+        return $random;
+    }
+
+    private function getRandomObject($em, $objectType)
+    {
+        $data = $em->getRepository($objectType)->findAll();
+        shuffle($data);
```

Comment:
neda shuffle za kazdym to iste? teda, ze ked to 'zamiesas' x-ty krat, tak to bude presne tak usporiadane, ako keby si to znova x-ty krat volal?

Reply:
shuffle ti nahodne pomiesa pole, takze by to malo byt ok